### PR TITLE
[DB] Remove redundant f-strings and validate ints

### PR DESCRIPTION
**Pull request by cjllanwarne**
Security Impact: No exploits possible, but this does make us look slightly better, and removes a small risk of incorrect component re-use in the future.

### CODE REVIEW

**Review by patrick-schultz — APPROVED** [review]
Thanks Chris! Obviously non-functional changes, but nice tidying up

File: batch/sql/insert_attempt_resources_format_version_lt_3.py [hunk 1]
```diff
@@ -70,7 +70,9 @@ async def _process(tx):
                               end_batch_id, end_batch_id, end_job_id, end_batch_id, end_job_id, end_attempt_id)
 
             if size is not None:
-                limit = f'LIMIT {size}'
+                # No risk in this file because all calls use int sizes,
+                # but this is good practice if we copy/paste this in the future:
+                limit = f'LIMIT {int(size)}'
```
Comment (chrisvittal) [inline batch/sql/insert_attempt_resources_format_version_lt_3.py:75]: I think this _theoretically_ could be parametrized. From [docs](https://dev.mysql.com/doc/refman/8.0/en/select.html):
> * Within prepared statements, `LIMIT` parameters can be specified using `?` placeholder markers.
This however is still safe.

**Review by chrisvittal — APPROVED** [review]
No functional changes. Just clearer intentions.

**Comment by cjllanwarne** [comment]
Closing because 
1. The change is purely cosmetic
2. Breaking previously-run migration checksums feels like a bad idea

**Comment by ehigham** [comment]
> Closing because
> 
> 1. The change is purely cosmetic
> 2. Breaking previously-run migration checksums feels like a bad idea

Do python files contribute to migration checksums?

**Pull request closed without merging**